### PR TITLE
add 203 status code - Non-Authoritative Information

### LINKED ISSUE #87: Missing image for 203 status code
Hey! 👋

The image for code (HTTP) 203 Non-Authoritative Information is missing

### PR DESCRIPTION
**Pull request by luizcieslak**
closes #87.

(Just want to mention that I love this website and use it often, thank you!)

### CODE REVIEW

File: src/lib/statuses.js [hunk 1]
```diff
@@ -48,7 +49,7 @@ const statuses = {
   451: { code: 451, message: 'Unavailable For Legal Reasons' },
   499: { code: 499, message: 'Client Closed Request' },
   500: { code: 500, message: 'Internal Server Error' },
-  501: { code: 501, message: 'Not Implemented'},
+  501: { code: 501, message: 'Not Implemented' },
```
Comment (luizcieslak) [inline src/lib/statuses.js:52]:
```suggestion
  501: { code: 501, message: 'Not Implemented'},
```

File: src/lib/statuses.js [hunk 2]
```diff
@@ -58,7 +59,7 @@ const statuses = {
   509: { code: 509, message: 'Bandwidth Limit Exceeded' },
   510: { code: 510, message: 'Not Extended' },
   511: { code: 511, message: 'Network Authentication Required' },
-  599: { code: 599, message: 'Network Connect Timeout Error' }
+  599: { code: 599, message: 'Network Connect Timeout Error' },
```
Comment (luizcieslak) [inline src/lib/statuses.js:62]:
```suggestion
  599: { code: 599, message: 'Network Connect Timeout Error' }
```

**Comment by rogeriopvl** [comment]
Hi @luizcieslak 

Looks great! Can you just change the font in the image to match the one we're using? (I don't know from the top of my head what font we're using, but I can find out).

**Comment by luizcieslak** [comment]
@rogeriopvl I was using `Libre Franklin` font before, now I'm using `Times` and it looks much close to the others. Although I think that the font used in the text is different from the one used in the status number. Let me know what u think or if u remember the correct ones!

**Comment by luizcieslak** [comment]
and if `Times` is OK I will update #113 as well.

**Comment by rogeriopvl** [comment]
Yes, the Times font works! Thanks! Just a couple of small issues that I also found:

- the white border around the image should be thinner (probably 50%)
- The image seems a bit low res/blurry around the text part, can it be improved?

**Comment by luizcieslak** [comment]
@rogeriopvl fixed! I was giving it too much compression, much better now. Update #113 too

**Comment by rogeriopvl** [comment]
Looks great! Thanks @luizcieslak , I'm going to merge both PRs and add you to the credit list on README

**Comment by rogeriopvl** [comment]
@all-contributors add @luizcieslak for content

**Pull request merged**